### PR TITLE
Euler2

### PR DESCRIPTION
**Pull request by gparsloe**
Euler 2 ready for review.

### CODE REVIEW

**Comment by gparsloe** [comment]
Hey Trevor - Forgot I had made a previous 'initial' commit. Oopsies...

File: 2GP.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    while (fibTermB <= 4000000){
+        if (fibTermB %2 === 0){
+            accumulator = accumulator + fibTermB;
+        }
+        sumTempHolder = fibTermA + fibTermB;
+        fibTermA = fibTermB;
+        fibTermB = sumTempHolder;
+    }
+    console.log("Sum off even fib terms to 4 million: " + accumulator);
```
Comment (gotrevor) [inline 2GP.js]: s/off/of/

**Review by gotrevor — CHANGES_REQUESTED** [review]

File: 2GP.js (hunk 1 above)
Comment (gparsloe) [reply 2GP.js]: Huh? I don't understand s/off/of/

Comment (gparsloe) [reply 2GP.js]: Oh.... sum of (not sum 'off')... ok... will change and re-commit.

Comment (gparsloe) [reply 2GP.js]: Changed sum off to sum of.

Comment (gparsloe) [reply 2GP.js]: Github is really confusing in terms of where to put comments.

Comment (gotrevor) [reply 2GP.js]: That `s/a/b/` is `sed` syntax.  Also used in bunches of other places to indicate "substitute `a` with `b`"

**Review by gotrevor — APPROVED** [review]
The linter I'll set up eventually will require 2 space indentation.  It will be the AirBnb eslint configuration, which is quite popular.

**Review by gparsloe — COMMENTED** [review]
Ok. Thanks.

**Pull request merged**